--- M2Crypto-0.23.0/tests/test_ssl.py	2015-12-04 18:12:27.000000000 +0200
+++ M2Crypto-0.23.0-patched/tests/test_ssl.py	2016-03-09 12:00:12.370781411 +0200
@@ -410,6 +410,7 @@
     # TLS is required in FIPS mode
     @unittest.skipIf(fips_mode, "Can't be run in FIPS mode")
     @unittest.skipIf(plat_debian, "Debian distros don't allow weak ciphers")
+    @unittest.skip("No weak crypto allowed, so skipping test")
     def test_sslv23_weak_crypto(self):
         self.args = self.args + ['-ssl2']
         pid = self.start_server(self.args)
@@ -461,6 +462,7 @@
 
     # TLS is required in FIPS mode
     @unittest.skipIf(fips_mode, "Can't be run in FIPS mode")
+    @unittest.skip("No weak crypto allowed, so skipping test")
     def test_no_weak_cipher(self):
         self.args = self.args + ['-cipher', 'EXP']
         pid = self.start_server(self.args)
@@ -479,6 +481,7 @@
     @unittest.skipIf(plat_debian, "Debian distros don't allow weak ciphers")
     @unittest.skipIf(plat_fedora,
                      "Export ciphers are prohibited in recent Fedora releases")
+    @unittest.skip("No weak crypto allowed, so skipping test")
     def test_use_weak_cipher(self):
         self.args = self.args + ['-cipher', 'EXP']
         pid = self.start_server(self.args)
